src/main/webapp/registration.html

```diff
@@ -4,7 +4,7 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Givr</title>
-	<link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="style.css">
```

Comment:
you should fix the indentations here

Reply:
I think we should fix all of the spacing (especially in the HTML files) in the Lint-Only PR!

Reply:
I think you should revert this part of the change? It looks ilke it was correct before this?